File: java/commands/Ping.java
package commands;
import structures.Discord;
import structures.Command;
import structures.CommandInfo;
import structures.CommandInput;

public class Ping extends Command {
    public Ping(Discord discord, CommandInput input) {
        super(discord, input);
        this.info = new CommandInfo("ping", "Pong");
    }

    public void run(String[] args) {
        input.reply("Pong").queue();
    }
}

File: c#/commands/Ping.cs
using Discord.WebSocket;
using Structures;

namespace Commands {
    public class Ping : ICommand {
        private DiscordClient discord;
        private SocketMessage message;

        public Ping(DiscordClient discord, SocketMessage message) {
            this.discord = discord;
            this.message = message;
        }

        public async Task Run(string[] args) {
            await message.Channel.SendMessageAsync("Pong");
        }
    }
}